src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -19,6 +19,7 @@ use rustc_ast::ast::{self, Block, ForeignItem, ForeignItemKind, Item, ItemKind,
 use rustc_ast::ast::{AssocItem, AssocItemKind, MetaItemKind, StmtKind};
 use rustc_ast::token::{self, Token};
 use rustc_ast::visit::{self, AssocCtxt, Visitor};
+use rustc_ast_lowering::Resolver as ResolverTrait;
```

Comment:
There are two `Resolver` traits actually, one in `librustc_ast_lowering` and one in `librustc_expand`.
So, I'd prefer to keep things as is in this PR and then perhaps rename the traits to `ResolverExpand` and `ResolverAstLowering` in a separate PR.

Reply:
Ok but i need to bring the trait into scope in order to make `local_def_id` method available.
And because there is already a structure named `Resolver`, I had to rename it somehow.
I can change the import line to:
```Rust
use rustc_ast_lowering::Resolver as _;
```
if you think that's better.

Reply:
Ah, ok, `... as ResolverAstLowering` then.

Reply:
Fixed.